README.md

```diff
@@ -2,58 +2,63 @@
 
 # Project Name
 
-> One paragraph statement about the project.
+> Project Microverse Trials Code Review
 
-![screenshot](./app_screenshot.png)
+![screenshot](norman.png)
 
-Additional description about the project and its features.
+This exercise was made to us make some improvements to your trial code, use the linter tool to check the source, and for us training how to be a code reviewer.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages: HTML
+- Frameworks: Visual Studio
+- Technologies used: Visual Studio, HTML, Linter Code check, bash.
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://leonardoeng13.github.io/review-trials2/)
```

Comment:
Great to know you found an answer for the live demo. Always remember to access the repo settings and enable the function pages. Otherwise, your live demo will not work.